app/assets/stylesheets/components.scss

```diff
@@ -168,6 +168,157 @@
   margin-bottom: 15px;
 }
 
+.project-cards-grid {
+  display: grid;
+  gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 396px));
+  justify-content: center;
+  margin: 20px auto;
+  width: 100%;
+}
+
+.project-card {
+  background: $white;
+  border: 1px solid #e5e5e5;
```

Comment:
use variables

Reply:
resolved in latest commit, please review again